src/main/resources/application.yml

```diff
@@ -43,7 +43,7 @@ spring:
       validationTimeout: 500
 
   flyway:
-    locations: classpath:/migration/postgres,classpath:/migration/common
+    locations: classpath:/migration/common,classpath:/migration/triggers
```

Comment:
postgres folder does not exist... Added a new triggers folder which isn't called in H2 for integration tests, because H2 does not support triggers

Reply:
I still wonder about triggers. If its just dates, these can be defaulted or set in the service? Its the same reasons for avoiding DB-specific features like JSON columns.